jupyter_server/services/sessions/sessionmanager.py

```diff
@@ -268,15 +286,51 @@ async def create_session(
         self._pending_sessions.remove(record)
         return result
 
-    def get_kernel_env(self, path):
-        """Return the environment variables that need to be set in the kernel"""
+    async def get_kernel_env(
+        self, path: Optional[str], name: Optional[ModelName] = None
+    ) -> Dict[str, str]:
+        """Return the environment variables that need to be set in the kernel
+
+        path : str
+            the path for the given session – seem to be a session id sometime.
```

Comment:
Can we please try to avoid this kind of conjecture?  
```suggestion
            the path for the given session
```
@blink1073 - I believe you mentioned the setting of `path` to a UUID was related to the pending-kernels stuff.  Is path == UUID now unconditional?  Does it _eventually_ become the true path?  Are you able to provide a definitive description of the `path` parameter?

Reply:
It will stay as a UUID for a JS console, unless we make consoles file-backed.  I think we can say it is the URL path.

Reply:
```suggestion
            the url path for the given session.
```